runtime/modules/check_version.lua

```diff
@@ -0,0 +1,16 @@
+local MIN_VERSION = "0.9.0"
+
+vim.api.nvim_create_autocmd({ "UIEnter" }, {
+  group = vim.api.nvim_create_augroup("VSCodeCheckVersion", {}),
+  callback = function()
+    local cur = vim.version()
+    local min = vim.version.parse(MIN_VERSION)
```

Comment:
To support this on older Nvim versions, just check if `vim.version.parse` exists. And if it doesn't, use the old vimscript api to call `vscode.window.showErrorMessage` (or send a request manually via `rpcnotify()`).